scss/civicrm/contact/_contact.scss

```diff
@@ -1,7 +1,13 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
 #{civi-page('contact-view')} {
   @import 'tabs';
   @import 'header';
   @import 'detail';
+
+  .select2-search::before {
```

Comment:
why is this needed only for this page?

Reply:
This page has `margin-top: 8px` on `input[type='text']` so, it needs counter css for fixing search icon position and make it look proper.
https://github.com/civicrm/org.civicrm.shoreditch/blob/master/scss/civicrm/contact/_detail.scss#L779-L784

Screenshot:
![image](https://user-images.githubusercontent.com/26058635/36665461-a3802888-1b0d-11e8-8690-3fb840b13ade.png)

Reply:
> This page has `margin-top: 8px` on `input[type='text']`

why?

Reply:
All popups which open in Contact view popups are styled that way.
There are many popups such as below having such style. It is 2 year old styling.
![image](https://user-images.githubusercontent.com/26058635/36666088-8b2a88a8-1b0f-11e8-87e1-17b9b109a71d.png)

I just moved this style from common/_forms.scss to this page specific now. In near future, when I work on popups, this will be removed and no longer needed.